src/Common/LRUCache.h

```diff
@@ -22,13 +21,32 @@ struct TrivialWeightFunction
     }
 };
 
+template <typename T>
+struct TrivialLRUCacheEvitPolicy
+{
+    // To note that the arg could be null
```

Comment:
I think we can remove this comment (and the same below), because it is more like implementation detail you used.